packages/grafana-ui/src/utils/storybook/UseState.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 interface StateHolderProps<T> {
   initialState: T;
-  children: (currentState: T, updateState: (nextState: T) => void) => JSX.Element;
+  children: (currentState: T, updateState: (nextState: T) => void) => React.ReactNode;
```

Comment:
I had an issue for some time with this in the story. I changed it later so it is not needed but I think expecting any ReactNode makes more sense here than just JSX.Element.